### PR TITLE
add missing DigiCert intermediate certificate to installations (DOS-883)

### PR DESCRIPTION
**Pull request by nadeemshahzad**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -8,6 +8,16 @@
...
     - "install:configuration"
 
+- stat:
+    path: "/usr/local/share/ca-certificates"
+  register: download_digcert_ca_certificate
+
+- name: Download digicert intermediate Certificate
+  get_url:
+    url: "{{ common_digicert_base_url }}/{{ common_digicert_name }}.pem"
+    dest: "/usr/local/share/ca-certificates/{{ common_digicert_name }}"
```
Comment (adeelehsan) [inline playbooks/roles/common/tasks/main.yml:22]: Please remove `.pem` after downloading the file. We need `.crt` file to add it in `ca-certificate.cert`.  
e.g `mv DigiCertSHA2SecureServerCA.crt.pem DigiCertSHA2SecureServerCA.crt`

Comment (nadeemshahzad) [reply playbooks/roles/common/tasks/main.yml:22]: We are downloading the file with ```.crt``` extension.

Comment (adeelehsan) [reply playbooks/roles/common/tasks/main.yml:22]: Actually we have to download the file in `.pem` and then change its  extension to `.cert`

**Review by adzuci — COMMENTED** [review]
Is there a way to test this logic by manually running the playbook and triggering an ID verification in a test environment?

**Comment by nadeemshahzad** [comment]
@adzuci We have build sandbox https://dos-883.sandbox.edx.org/ and Adeel Ehsan has verified that that software secure https link is working and we are not facing any cert issue.

**Review by adzuci — APPROVED** [review]
:shipit: As long as the sandbox looks good.

**Pull request merged**